src/components/icon/icon.stories.js

```diff
@@ -41,6 +43,8 @@ function dlsKnobs() {
   return {
     bgTheme,
     fontSize,
+    ml: number('ml', 0),
+    mr: number('mr', 0),
```

Comment:
The code sample in the `classic` Story Source isn't showing these new props, but the Props Table is
<img width="885" alt="Screenshot 2020-09-02 at 14 25 45" src="https://user-images.githubusercontent.com/35449967/91989432-6d830800-ed28-11ea-87ee-acb0beb110fe.png">

Reply:
It's because the classic story doesn't have the knobs for mr/ml, only the `default` story. IMO no need to add them to the classic story as we'll be removing it imminently.